recipes/django_polymorphic/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set version = "0.7.1" %}
+
+package:
+    name: django_polymorphic
+    version: {{ version }}
+
+source:
+    fn: django_polymorphic-{{ version }}.tar.gz
+    url: https://pypi.python.org/packages/source/d/django_polymorphic/django_polymorphic-{{ version }}.tar.gz
+    md5: c6999aa8aa4d5a992e2664f050a8e858
+
+build:
+    number: 0
+    script: python setup.py install --single-version-externally-managed --record=record.txt
+    preserve_egg_dir: True
+
+requirements:
+    build:
+        - python
+        - setuptools
+        #- django
+    run:
+        - python
+        #- setuptools
+        - django
+
+#test:
+    #requires:
+        #- coverage
+    #files:
+        #- runtests.py
+    #commands:
+        #- cp runtests.py $SRC_DIR && cd $SRC_DIR && coverage run --source=polymorphic runtests.py
```

Comment:
Not sure the linter will let a recipe with a test pass, but we are looking into relaxing that rule anyway.